fast64_internal/oot/oot_collision_classes.py

```diff
@@ -93,65 +93,65 @@
     ("CAM_SET_DUNGEON0", "Dungeon0", "Dungeon0"),
     ("CAM_SET_DUNGEON1", "Dungeon1", "Dungeon1"),
     ("CAM_SET_NORMAL3", "Normal3", "Normal3"),
-    ("CAM_SET_HORSE0", "Horse0", "Horse0"),
-    ("CAM_SET_BOSS_GOMA", "Boss_goma", "Boss_goma"),
-    ("CAM_SET_BOSS_DODO", "Boss_dodo", "Boss_dodo"),
-    ("CAM_SET_BOSS_BARI", "Boss_bari", "Boss_bari"),
-    ("CAM_SET_BOSS_FGANON", "Boss_fganon", "Boss_fganon"),
-    ("CAM_SET_BOSS_BAL", "Boss_bal", "Boss_bal"),
-    ("CAM_SET_BOSS_SHADES", "Boss_shades", "Boss_shades"),
-    ("CAM_SET_BOSS_MOFA", "Boss_mofa", "Boss_mofa"),
-    ("CAM_SET_TWIN0", "Twin0", "Twin0"),
-    ("CAM_SET_TWIN1", "Twin1", "Twin1"),
-    ("CAM_SET_BOSS_GANON1", "Boss_ganon1", "Boss_ganon1"),
-    ("CAM_SET_BOSS_GANON2", "Boss_ganon2", "Boss_ganon2"),
-    ("CAM_SET_TOWER0", "Tower0", "Tower0"),
-    ("CAM_SET_TOWER1", "Tower1", "Tower1"),
-    ("CAM_SET_FIXED0", "Fixed0", "Fixed0"),
-    ("CAM_SET_FIXED1", "Fixed1", "Fixed1"),
-    ("CAM_SET_CIRCLE0", "Circle0", "Circle0"),
-    ("CAM_SET_CIRCLE2", "Circle2", "Circle2"),
-    ("CAM_SET_CIRCLE3", "Circle3", "Circle3"),
-    ("CAM_SET_PREREND0", "Prerend0", "Prerend0"),
-    ("CAM_SET_PREREND1", "Prerend1", "Prerend1"),
-    ("CAM_SET_PREREND3", "Prerend3", "Prerend3"),
+    ("CAM_SET_HORSE", "Horse", "Horse"),
```

Comment:
the first item in each tuple is the identifier for the enum item, it can't be changed without breaking older .blend files unfortunately. The rest is fine to change as you wish

For more information see https://docs.blender.org/api/current/bpy.props.html?highlight=items#bpy.props.EnumProperty

So to update these names we would need a identifier -> decomp name mapping to use after `getCustomProperty(camPosProp, 'camSType')`

@ me on the fast64 discord or DMs if needed

Reply:
@Dragorn421 Oh didn't consider that. Updated in f4065cb with your suggestion.